src/simulation/m_acoustic_src.fpp

```diff
@@ -108,7 +111,12 @@ contains
                 delay(i) = acoustic(i)%delay
             end if
         end do
-        !$acc update device(loc_acoustic, mag, dipole, support, length, height, wavelength, frequency, gauss_sigma_dist, gauss_sigma_time, foc_length, aperture, npulse, pulse, dir, delay, element_polygon_ratio, rotate_angle, element_spacing_angle, num_elements, element_on, bb_num_freq, bb_bandwidth, bb_lowest_freq)
+        $:GPU_UPDATE(device=["loc_acoustic","mag","dipole","support","length", &
```

Comment:
i created an fypp macro that works so that you don't have to create a pythonic list for these (i think). it's a bit more simple (less quotes!).

check out https://github.com/MFlowCode/MFC/issues/834 and see how i call it as

```
    $:mymacro(arg1="[1,2]", arg3="[1,a]")
```